tasks/collections/build.js

```diff
@@ -2,17 +2,15 @@
  *    Define Build Sub-Tasks
  *******************************/
 
-module.exports = function (gulp) {
-    // build sub-tasks
-    const
-        watch       = require('../watch'),
+const watch = require('../watch');
 
-        build       = require('../build'),
-        buildJS     = require('../build/javascript'),
-        buildCSS    = require('../build/css'),
-        buildAssets = require('../build/assets')
-    ;
+// build sub-tasks
+const build = require('../build');
```

Comment:
Why did you put each const into a separate `const` statement instead of a comma separted list? in admin.js above, you did it as a comma separated list (most probably because of copy/paste form the origin)
I like the comma separated list instead (which seems to be done all the other times in the scripts already.
Let's be consistent and also have a comma separated list here 🙂

Reply:
I would say single declaration per variable is better for diff, readability, ... But definitely not to be changed here nor now, probably to be changed in #2584. Here fixed.